xskillscore/core/np_deterministic.py

```diff
@@ -55,7 +55,10 @@ def _check_weights(weights):
     return None to guide weighting scheme.
     """
     if weights is None:
-        return weights
+        return None
+    # catch if np.ndarray None
+    elif (weights == None).all():
```

Comment:
this raises a linter error:
` E711 comparison to None should be 'if cond is None:'`

somehow weights can be `None` when printed but of class `np.ndarray`. Therefore I use this check.

Ignoring E711?

Reply:
if anyone has an idea how to solve this, please comment here.

The problem: sometimes (and I dont understand when this is) weights is of type np.ndarray and None. to catch this I have this line. the suggested `if weights is None` is already above and doesnt catch these cases.

Reply:
my last commit ignores E711